webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVReassignActivationKeysForm.js

```diff
@@ -72,6 +76,14 @@ const CVReassignActivationKeysForm = () => {
     contentViewsInEnvError, selectedEnvForAK, setSelectedCVForAK, setSelectedCVNameForAK,
     cvInEnvLoading, selectedCVForAK, cvId, versionEnvironments, selectedEnvSet]);
 
+  useDeepCompareEffect(() => {
+    if (activationKeysResponse?.results) {
+      const hasMultiCVEnv = activationKeysResponse.results.some(key =>
+        key.multi_content_view_environment);
+      setMultiCVWarning(hasMultiCVEnv);
+    }
+  }, [activationKeysResponse]);
+
```

Comment:
This doesn't need a useState or a `useDeepCompareEffect`. It still seems to work fine if we just
```js
const multiCVWarning = activationKeysResponse?.results?.some?.(key =>
        key.multi_content_view_environment);
```

Reply:
Ack... Will implement the change

Reply:
@jeremylenz I have incorporated the suggested changes and pushed the code.